modules/ROOT/pages/_partials/showcase-apps/security.adoc

```diff
@@ -0,0 +1,20 @@
+= Security Services
+
+== {device-security-service}
+
+
+From the showcase app:
+
+. Press the *Security* menu item.
+. Press the *Device Trust* sub menu item.
+. View the results of the Device security checks.
+. Navigate to the {grafana-ui}, and view the metrics as described in xref:mobile-metrics.adoc[Analyzing App Usage]
+
+== {app-security-service}
+
+From the showcase app:
+
+. Press the *Security* menu item.
+. Press the *Application Trust* sub menu item.
+. Navigate to the Mobile Security Service link to set the enable/disable message as described in xref:app-security.adoc[using the Mobile Security Service]
```

Comment:
Would it be possible to link directly to the relevant section of the page? This link sends the reader to the top of **Using the App Security Service**, and they will then have to find **Using the App Security Console**.

Also, the link says "using the Mobile Security Service", but this does not match the actual title of the page.

Reply:
Unfortunately not , limitation of  asciidoc , links with underscore in them don't work, see https://github.com/asciidoctor/asciidoctor/issues/625 . So our `_partials` directory the issue here. Tried some of the workarounds with no joy.

On your second point links don't need to match the title of the page they are going to.

Reply:
_partials is only used for includes, should never affect linking.
I'm not sure what the issue is @austincunningham , but reach out if you wanna discuss

Reply:
FYI: nothing in _partials is published, ie unless it's included in a file that exists in ../pages, no html is produced.

Reply:
That helps thanks @finp